python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/model-clients.ipynb

```diff
@@ -174,6 +180,58 @@
     "**NB the default usage response is to return zero values**"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Caching Wrapper\n",
```

Comment:
This section should come after the streaming and Azure Open AI subsections. 

I think for the example we can show how to use `diskcache` as that's what most people will be using. 

To add diskcache for building the doc we will need to include it in the doc dependencies in the core's `pyproject.toml` file.